examples/example.krun

```diff
@@ -73,3 +73,6 @@ TEMP_READ_PAUSE = 1
 
 # CPU pinning (off by default)
 #ENABLE_PINNING = False
+
+# Autoload rmsr kernel module (on by defualt)
+# AUTOLOAD_RMSR = True
```

Comment:
On Linux, it currently makes no sense not to load rsmr (as other code depends on it), so I suggest removing the knob. Less knobs is good!